### PR TITLE
fix(test): update argument checks for GitHub handler

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: universum/github_handler.py [hunk 1]
```diff
@@ -20,15 +21,37 @@ def define_arguments(argument_parser):
                                      help='Currently parsed from "x-github-event" header')
         argument_parser.add_argument('--payload', '-pl', dest='payload', metavar="GITHUB_PAYLOAD",
                                      help='<actual help coming later> leave "-" to read from stdin')
-        argument_parser.add_argument('--trigger-url', '-tu', dest='trigger_url', metavar="TRIGGER_URL",
+        argument_parser.add_argument('--trigger-url', '-tu', dest='trigger', metavar="TRIGGER_URL",
```
Comment (i-keliukh) [inline universum/github_handler.py:24]: Why you renamed the variable?

**Comment by i-keliukh** [comment]
Did you manually check how error messages look in the terminal?

File: universum/modules/vcs/github_vcs.py [hunk 2]
```diff
@@ -26,9 +26,14 @@ def define_arguments(argument_parser):
         parser = argument_parser.get_or_create_group("GitHub", "GitHub repository settings")
 
         parser.add_argument("--github-app-id", "-gta", dest="integration_id", metavar="GITHUB_APP_ID",
-                            help="GitHub application ID (real help coming soon)")
+                            help="GitHub application ID to use for check run report. Only GitHub App "
+                                 "can report a check run result! If you don't have an App for reporting purposes, "
+                                 "please don't use '--report-to-review'  with VCS GitHub")
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:31]: with VCS Github -> with Github

File: universum/modules/vcs/github_vcs.py [hunk 3]
```diff
@@ -26,9 +26,14 @@ def define_arguments(argument_parser):
...
         parser.add_argument("--github-private-key", "-gtk", dest="key", metavar="GITHUB_PRIVATE_KEY",
-                            help="Application private key file path")
+                            help="GitHub App private key for obtaining OAuth token. Pass raw key data via "
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:33]: This is not OAuth token.

File: universum/modules/vcs/github_vcs.py [hunk 4]
```diff
@@ -84,7 +89,9 @@ def define_arguments(argument_parser):
 
         parser.add_argument("--github-installation-id", "-gti", dest="installation_id",
                             metavar="GITHUB_INSTALLATION_ID",
-                            help="Calculated out of webhook payload (real help coming soon)")
+                            help="GitHub installation ID identifies GitHub App user. Can be retrieved from "
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:92]: In my opinion "GitHub App user" is not correct term.
Installation ID identifies specific app installation into user account or organization.

File: universum/modules/vcs/github_vcs.py (hunk 2 above)
Comment (i-keliukh) [reply universum/modules/vcs/github_vcs.py]: Still an issue

File: universum/modules/vcs/github_vcs.py [hunk 5]
```diff
@@ -26,18 +26,46 @@ def define_arguments(argument_parser):
         parser = argument_parser.get_or_create_group("GitHub", "GitHub repository settings")
 
         parser.add_argument("--github-app-id", "-gta", dest="integration_id", metavar="GITHUB_APP_ID",
-                            help="GitHub application ID (real help coming soon)")
-        parser.add_argument("--github-private-key", "-gtk", dest="key_path", metavar="GITHUB_PRIVATE_KEY",
-                            help="Application private key file path")
+                            help="GitHub application ID to use for check run report. Only GitHub App "
+                                 "can report a check run result! If you don't have an App for reporting purposes, "
+                                 "please don't use '--report-to-review'  with GitHub")
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:31]: 2 space characters

File: universum/modules/vcs/github_vcs.py [hunk 6]
```diff
@@ -63,11 +91,23 @@ def define_arguments(argument_parser):
 
         parser.add_argument("--github-installation-id", "-gti", dest="installation_id",
                             metavar="GITHUB_INSTALLATION_ID",
-                            help="Calculated out of webhook payload (real help coming soon)")
+                            help="GitHub installation ID identifies GitHub App issuer. Can be retrieved from "
+                                 "web-hook or obtained via REST API; in standard workflow should be "
+                                 "received from `universum githbu-handler`")
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:96]: "githbu"

File: universum/modules/vcs/github_vcs.py [hunk 7]
```diff
@@ -63,11 +91,23 @@ def define_arguments(argument_parser):
 
         parser.add_argument("--github-installation-id", "-gti", dest="installation_id",
                             metavar="GITHUB_INSTALLATION_ID",
-                            help="Calculated out of webhook payload (real help coming soon)")
+                            help="GitHub installation ID identifies GitHub App issuer. Can be retrieved from "
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:94]: "issuer" is not valid term in this context

File: universum/modules/vcs/github_vcs.py [hunk 8]
```diff
@@ -63,11 +92,23 @@ def define_arguments(argument_parser):
 
         parser.add_argument("--github-installation-id", "-gti", dest="installation_id",
                             metavar="GITHUB_INSTALLATION_ID",
-                            help="Calculated out of webhook payload (real help coming soon)")
+                            help="GitHub installation ID identifies specific app installation into user account "
+                                 "or organization.. Can be retrieved from web-hook or obtained via REST API; "
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:96]: Two dots

File: universum/modules/vcs/github_vcs.py [hunk 9]
```diff
@@ -26,18 +26,46 @@ def define_arguments(argument_parser):
...
+
+        utils.check_required_option(self.settings, "key", """
+                    GitHub App private key not specified.
+
+                    As a multiline variable, the private key cannot be passed via command line directly.
+                    Please store it in environment variable ("GITHUB_PRIVATE_KEY"), or enter through the stdin
+                    (pass '-' param value for redirection), or pass a filename starting with '@' character, absolute or
+                    relative starting from project root.
+
+                    For security reasons, `universum github-handler` DOES NOT pass private key to CI builds.
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:60]: Please remove this sentence.

File: universum/modules/vcs/github_vcs.py [hunk 10]
```diff
@@ -26,18 +26,46 @@ def define_arguments(argument_parser):
...
+                    
+                    For security reasons, `universum github-handler` DOES NOT pass App ID to CI builds. 
+                    Please specify the checkout id by using '--github-app-id' ('-gta')
+                    command line parameter or by setting GITHUB_APP_ID environment variable.
+                """)
+
+        utils.check_required_option(self.settings, "key", """
+                    GitHub App private key not specified.
+
+                    As a multiline variable, the private key cannot be passed via command line directly.
```
Comment (i-keliukh) [inline universum/modules/vcs/github_vcs.py:55]: This is not true. It can be passed.

File: universum/modules/vcs/github_vcs.py (hunk 5 above)
Comment (i-keliukh) [reply universum/modules/vcs/github_vcs.py:31]: Still 2 spaces

**Comment by k-dovgan** [comment]
> Did you manually check how error messages look in the terminal?
Some of them, yes. Well, they look as usual?

**Pull request merged**